core/flyout.js

```diff
@@ -742,3 +897,93 @@ Blockly.Flyout.terminateDrag_ = function() {
   Blockly.Flyout.startBlock_ = null;
   Blockly.Flyout.startFlyout_ = null;
 };
+
+/**
+ * Compute height of flyout.  Position button under each block.
+ * For RTL: Lay out the blocks right-aligned.
+ */
+Blockly.Flyout.prototype.reflowHorizontal = function() {
+  this.workspace_.scale = this.targetWorkspace_.scale;
+  var flyoutHeight = 0;
+  var margin = this.CORNER_RADIUS;
+  var blocks = this.workspace_.getTopBlocks(false);
+  for (var x = 0, block; block = blocks[x]; x++) {
```

Comment:
We prefer i to x, unless dealing with horizontal coordinates.  Blockly has a lot of x and y, so let's not use x to mean something else.